package.json

```diff
@@ -25,6 +25,8 @@
     "babel-loader": "^8.0.6",
     "babel-plugin-formatjs": "^10.2.6",
     "blueimp-md5": "^2.10.0",
+    "cldr-annotations-derived-modern": "^41.0.0",
+    "cldr-annotations-modern": "^41.0.0",
```

Comment:
Unless/until we start using these in production, they should go in the `devDependencies` section, not `dependencies`.